lib/challenge.js

```diff
@@ -60,7 +79,11 @@ var some_pet;
 
 var Pet;
 // Answer Starts Here
-
+var Pet = function(name, species, breed, nosie)
+  this.name = name;
+  this.species = species;
+  this.breed = breed;
+  this.noise = makeNoise;
```

Comment:
This constructor function is well-formed except for the missing curly braces. The function body (where the statements that get run are) must be surrounded by curly braces. The assignments here are good, and usage of parameters is mostly good -- there is no `makeNoise` parameter taken by this function, but that looks more like a typo than anything. Finally, the requested prototype method is missing.